liquibase-core/src/main/resources/www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.17.xsd

```diff
@@ -81,6 +81,36 @@
             <xsd:enumeration value="QUOTE_ONLY_RESERVED_WORDS"/>
         </xsd:restriction>
     </xsd:simpleType>
+    <!-- include -->
```

Comment:
I am surprised that you modified the 4.17 xsd. Isn't this one already released, and so shouldn't be modified?

Reply:
@wwillard7800 Should this xsd change be only in latest.xsd?

Reply:
Yes, that is correct.  I will revert that change.